vivisect/qt/funcgraph.py

```diff
@@ -489,6 +490,18 @@ def renderFunctionGraph(self, fva, graph=None):
         #self.parentWidget().delEventCore(self)
         #return e_mem_qt.VQMemoryWindow.closeEvent(self, event)
 
+    def _getLocVa(self, addr):
```

Comment:
So I like this function, but it feels like it doesn't belong here, and duplicates `getLocations` from __init__.py

I'd say either add an optional parameter to `getLocation` (singular version) that filters on location type, or just use `getLocations` and pass the location type into their. Feels cleaner that way.